package-lock.json

```diff
@@ -11895,7 +11895,7 @@
     },
     "oidc-client": {
       "version": "1.4.1",
-      "resolved": "https://registry.npmjs.org/oidc-client/-/oidc-client-1.4.1.tgz",
```

Comment:
Registry should be: registry.npmjs.org

Reply:
Put back registry.npmjs.org for registry